lib/actions/api.js

```diff
@@ -563,7 +649,7 @@ function createQueryAction (endpoint, responseAction, errorAction, options) {
     }
     let payload
     try {
-      const response = await fetch(url)
+      const response = await fetch(url, options.fetchOptions)
```

Comment:
These options will mess a bit with the fetch throttling I implemented in the micromobility branch.